### PR TITLE
MM-21956 Updated UX

### PR DESCRIPTION
**Pull request by catalintomai**
Updated Group Sync To Roles Chat facing UI (Teams and Channels Modals)

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-21956

### CODE REVIEW

File: components/admin_console/manage_teams_modal/manage_teams_dropdown.jsx [hunk 1]
```diff
@@ -77,7 +77,7 @@ export default class ManageTeamsDropdown extends React.Component {
                     <Menu.ItemAction
                         show={isTeamAdmin}
                         onClick={this.makeMember}
-                        text={Utils.localizeMessage('admin.user_item.makeMember', 'Make Member')}
+                        text={Utils.localizeMessage('admin.user_item.makeMember', 'Make Team Member')}
```
Comment (sbishel) [inline components/admin_console/manage_teams_modal/manage_teams_dropdown.jsx]: Did you run `make i18n-extract`? I think this change will necessitate a change to ../i18n/en.json.

File: components/channel_groups_manage_modal/channel_groups_manage_modal.jsx [hunk 2]
```diff
@@ -90,7 +92,7 @@ class ChannelGroupsManageModal extends React.PureComponent {
                     height='32'
                 />
                 <div className='more-modal__details'>
-                    <div className='more-modal__name'>{item.display_name} {'-'} <span>
+                    <div className='more-modal__name'>{item.display_name} {'-'}&nbsp;<span className='more-modal__name_count'>
```
Comment (sbishel) [inline components/channel_groups_manage_modal/channel_groups_manage_modal.jsx]: nit: Add Linebreak and indent before <span> tag.

File: components/team_groups_manage_modal/team_groups_manage_modal.jsx [hunk 3]
```diff
@@ -117,22 +119,28 @@ class TeamGroupsManageModal extends React.PureComponent {
                     height='32'
                 />
                 <div className='more-modal__details'>
-                    <div className='more-modal__name'>{item.display_name} {'-'} <span>
+                    <div className='more-modal__name'>{item.display_name} {'-'} &nbsp;<span className='more-modal__name_count'>
```
Comment (sbishel) [inline components/team_groups_manage_modal/team_groups_manage_modal.jsx]: same nit.

File: components/admin_console/manage_teams_modal/manage_teams_dropdown.jsx (hunk 1 above)
Comment (catalintomai) [reply components/admin_console/manage_teams_modal/manage_teams_dropdown.jsx]: This is to fix the discrepancy between the string in i18n file and the default one used in tests.

**Comment by srkgupta** [comment]
Approving the PR on behalf of @michaelgamble as per confirmation from @catalintomai 
Screenshot 2020-01-27 at 5 37 25 PM

**Pull request merged**